src/gax/Cargo.toml

```diff
@@ -25,6 +25,12 @@ keywords.workspace   = true
 license.workspace    = true
 repository.workspace = true
 
+[package.metadata.docs.rs]
+# We want to generate documentation for streaming APIs, gated by this feature.
+# We do not want to generate documentation for `unstable-sdk-client`, which
```

Comment:
Maybe it should be called `_unstable-sdk-client`...

Reply:
Ack, not for this PR.